src/missed_spans.rs

```diff
@@ -94,9 +94,10 @@ impl<'a> FmtVisitor<'a> {
         fn replace_chars(string: &str) -> String {
             string.chars()
                   .map(|ch| {
-                      match ch.is_whitespace() {
-                          true => ch,
-                          false => 'X',
+                      if ch.is_whitespace() {
+                          ch
+                      } else {
+                          'X'
```

Comment:
Well, in a sense, `match` is shorter than `if else` here because rustfmt formats so...